### PR TITLE
fix: fw-1084 delete post in thread

### PR DESCRIPTION
**Pull request by swkatmask**
https://mask.atlassian.net/browse/fw-1084
https://mask.atlassian.net/browse/fw-1146

### CODE REVIEW

File: src/components/Actions/ReportUserButton.tsx [hunk 1]
```diff
@@ -24,12 +24,13 @@ export const ReportUserButton = forwardRef<HTMLButtonElement, Props>(function Re
             onClick={async () => {
                 rest.onClick?.();
                 const confirmed = await ConfirmModalRef.openAndWaitForClose({
-                    title: t`Report`,
+                    title: t`Report @${profile.handle}`,
                     content: (
                         <div className="text-main">
-                            <Trans>Confirm you want to report @{profile.handle}?</Trans>
+                            <Trans>Confirm to report this user</Trans>
```
Comment (guanbinrui) [inline src/components/Actions/ReportUserButton.tsx:30]:
```suggestion
                            <Trans>Confirm to report this user?</Trans>
```

Comment (swkatmask) [reply src/components/Actions/ReportUserButton.tsx:30]: image

There isn't a question mark in the figma design.

Comment (guanbinrui) [reply src/components/Actions/ReportUserButton.tsx:30]: It's a obvious mistake.

Comment (swkatmask) [reply src/components/Actions/ReportUserButton.tsx]: Updated.

**Pull request merged**